src/redux/books/Books.js

```diff
@@ -24,4 +28,47 @@ export const addBook = (payload) => ({
   payload,
 });
 
+export const listBooks = (payload) => ({
+  type: LIST_BOOK,
+  payload,
+});
+
+export const getBooks = async () => {
+  const response = await fetch(POST_BOOK);
+  const APIbooks = await response.json();
+  const books = Object.getOwnPropertyNames(APIbooks).map((id) => ({
+    item_id: id,
+    ...APIbooks[id][0],
+  }));
+  return (books);
+};
```

Comment:
- [ ] Nice work here, although this logic works properly, the function `getBooks` should return a function. That is, it ought to return a function with `dispatch` as its parameter. Kindly refactor the code a bit to suit this project requirement. 👍 

#### _**Hint:**_ `const getBooks = () => async(dispatch)=> {write code}`